packages/gcf-utils/package.json

```diff
@@ -62,4 +65,4 @@
   "files": [
     "build/src"
   ]
-}
+}
```

Comment:
newline to make the red squiggle go away please :)

Reply:
Is there a formatter that you use that doesn't do this? I've been using VSCode's `Format on save` feature and it always removes the newline at the end of file (which requires a bunch of dancing to disable and re-enable to fix this).

Reply:
Hmmm, not that I know of for JSON files.  We have it set up properly for prettier integration for ts and js files though.  Maybe I need to see if prettier does JSON....